content/1.getting-started/2.concept.md

```
@@ -0,0 +1,144 @@
+---
+title: Concept
+description: Learn the concept of Laravel Access Control
+---
+
+Laravel Access Control helps you secure your application in the same place. Secure queries and endpoints using the same logic !
+
+Here is a simple diagram of the concept:
+<img src="/diagram.png" />
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)